advanced/Scripts/webpage.sh

```diff
@@ -538,7 +538,8 @@ Interfaces:
 }
 
 Teleporter() {
-    local datetimestamp=$(date "+%Y-%m-%d_%H-%M-%S")
+    datetimestamp=$(date "+%Y-%m-%d_%H-%M-%S")
```

Comment:
Can you flip these two lines so that the variable is declared local before the value is assigned to it? Thanks!